<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaceForEncryption</class>
 <widget class="QWidget" name="interfaceForEncryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="hash_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_key">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="encrypt_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Зашифровавать </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="return_init">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>return to init window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
